lib/oli_web/live/sections/gating_and_scheduling.ex

```diff
@@ -83,16 +79,12 @@ defmodule OliWeb.Sections.GatingAndScheduling do
           {Gating.get_gating_condition!(int_id), "Student Exceptions"}
       end
 
-    case Mount.for(section_slug, session) do
-      {:admin, _author, section} ->
-        {:ok, assign_defaults(socket, section, session, parent_gate, title)}
+    {user_type, _user, section} = Mount.for(section_slug, session)
```

Comment:
Should we cover from the case when this function returns an `:error`? Same for the other function calls

Reply:
Definitely 👍🏼